uber/models.py

```diff
@@ -1938,6 +1953,22 @@ class ArbitraryCharge(MagModel):
     _repr_attr_names = ['what']
 
 
+class LedgerItem(MagModel):
+    ident = Column(UnicodeText, nullable=True)
```

Comment:
Hm, I guess it's not clear. For right now this stores the transaction ID from Stripe.

Since I'll probably reduce this scope of this down (under YAGNI principles) to being just a 'stripe transaction history table,' I can rename this to something less generic and more informative, like `stripe_id`.